service/migration_manager.cc

```diff
@@ -1170,7 +1169,6 @@ future<> migration_manager::on_change(gms::inet_address endpoint, gms::applicati
 }
 
 future<> migration_manager::on_alive(gms::inet_address endpoint, gms::endpoint_state state) {
-    schedule_schema_pull(endpoint, state);
```

Comment:
Where is "if raft is disabled"?

Why is the patch changelog different from the cover letter?

Reply:
Sorry, forgot to mention I wanted to make it fail on CI. Last push is not the patch.